XenModel/XenAPI/FriendlyErrorNames.resx

```diff
@@ -1959,7 +1959,7 @@ Authorized Roles: {1}</value>
     <value>The VGPU is not compatible with any PGPU in the destination.</value>
   </data>
   <data name="VGPU_TYPE_NOT_COMPATIBLE" xml:space="preserve">
-    <value>You try to create a VGPU that is not compatible with existing types on the VM.</value>
+    <value>You tried to create a VGPU that is not compatible with existing types on the VM.</value>

```

Comment:
We should avoid using "VGPU" in the user-facing messages, so here we should say "virtual GPU" .
My suggestion for this message was (note "virtual GPU" and "the existing types"):
"You tried to create a virtual GPU that is not compatible with the existing types on the VM."

Reply:
I've submit a PR in XAPI to update the text.